### PR TITLE
WIP: Generate clientset for BackupDriver APIs

### PR DESCRIPTION
**Pull request by xing-yang**
This PR generates clientset and informers for BackupDriver APIs.

### CODE REVIEW

File: pkg/generated/crds/manifests/backupdriver.io_backuprepositories.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  creationTimestamp: null
+  name: backuprepositories.backupdriver.io
+spec:
+  group: backupdriver.io
+  names:
+    kind: BackupRepository
+    listKind: BackupRepositoryList
+    plural: backuprepositories
+    singular: backuprepository
+  scope: Namespaced
```
Comment (swgupta-vmw) [inline pkg/generated/crds/manifests/backupdriver.io_backuprepositories.yaml:17]: Should BackupRepository be cluster-scoped?

File: pkg/apis/backupdriver/v1/backup_repository.go [hunk 2]
```diff
@@ -20,9 +23,24 @@ type BackupRepository struct {
 
 	RepositoryDriver      string            `json:"repositoryDriver"`
 	RepositoryParameters  map[string]string `json:"repopsitoryParameters"`
-	BackupRepositoryClaim string            `json:backupRepositoryClaim`
+	BackupRepositoryClaim string            `json:"backupRepositoryClaim"`
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// BackupRepositoryList is a list of BackupRepositoryClaim resources
```
[the remark this answers is not in the document]
Comment (xing-yang) [reply pkg/apis/backupdriver/v1/backup_repository.go]: This is needed for any api object.

File: pkg/generated/crds/manifests/backupdriver.io_backuprepositories.yaml (hunk 1 above)
Comment (xing-yang) [reply pkg/generated/crds/manifests/backupdriver.io_backuprepositories.yaml:17]: Yes.  Fixed.

**Comment by xing-yang** [comment]
Closing this one.  Will submit another PR to fix the make error.

**Comment by xing-yang** [comment]
New PR is submitted here: https://github.com/vmware-tanzu/velero-plugin-for-vsphere/pull/88

**Pull request closed without merging**